pkg/base/monitor/wmiEventMonitor.go

```diff
@@ -152,6 +153,8 @@ func (n *WmiNotificationMonitor) Start() error {
 func (n *WmiNotificationMonitor) handleNotification(wmiEventMessage WmiEventMessage) {
 	ctx := context.Background()
 
+	log.Printf("Received a WMI Event: %s", wmiEventMessage.ToString())
```

Comment:
This would create a lot of noise and cannot commit. Can you use this change for private testing?

Reply:
Sure, makes sense. This could overwhelm the log files in the very later future. Removed the generic ones.
I have only kept for the error condition if it happens and will definitely be helpful